blog/_posts/2017-03-01-zinc-blog-post.md

```diff
@@ -0,0 +1,141 @@
+can ship in any build tool supporting Scala.
+  
+## Releasing Zinc 1.0.0-X10
+  
+The experimental release includes previous Ligthbend's improvements to the incremental
+algorithm, contributions from several OSS contributors (like [Stu Hood](https://github.com/sbt/zinc/pulls?utf8=✓&q=is:pr%20is:closed%20author:stuhood),
+[Krzysztof Romanowski](https://github.com/sbt/zinc/pulls?utf8=✓&q=is:pr%20is:closed%20author:romanowski) and
+[Peiyu Wang](https://github.com/sbt/zinc/pulls?q=is:pr+author:peiyuwang+is:closed)) and
+and the following contributions from the Scala Center:
+  
+* [Improve binary API and documentation](https://github.com/sbt/zinc/pull/236).
+* [Fix handling of As Seen From and analysis in structural types](https://github.com/sbt/zinc/pull/239).
+* [Fix handling of local dependencies for java classes](https://github.com/sbt/zinc/pull/217).
+* [Implement SafeLazy in Java](https://github.com/sbt/zinc/pull/221).
+* [Add JMH suite to benchmark any project in the
+  community and improve Zinc's performance](https://github.com/sbt/zinc/pull/225).
+* Fix [several](https://github.com/sbt/zinc/pull/21://github.com/sbt/zinc/pull/219) [infrastructure](https://github.com/sbt/zinc/pull/228) [issues](https://github.com/sbt/zinc/pull/232).
+  
+All in all, a part of these changes address all the missing tasks that [Grzegorz
+documented as missing](https://github.com/sbt/zinc/issues/174), and the rest
+improve the quality of the project. With also more insight into how to solve the
+current problems, the Scala Center has spent time figuring out how to address
+well-known issues of the incremental compiler, like [handling of changes to
+final constants](https://github.com/sbt/zinc/issues/227), [private members in
+traits](https://github.com/sbt/zinc/issues/118) and [stub errors
+handling](https://github.com/sbt/zinc/issues/150). The purpose of this work has
+been to improve even more the correctness of the compiler. In the next days, the
+Zinc team will provide an implementation for the fixes.
+  
+## Trying out Zinc 1.0
+  
+Thanks to class-based dependency and name-hashing heuristics, Zinc is smarter
+when it comes to detecting dependencies between source files. For instance,
+complex patterns like the [cake
+pattern](http://www.cakesolutions.net/teamblogs/2011/12/19/cake-pattern-in-depth),
+among other situations, are now efficiently supported.
+  
+In this example, we're going to try Zinc 1.0 to compile
+[ScalaTest](http://www.scalatest.org/). We're going to add a method to a widely
+referenced class: `AndHaveWord`, in file `Matcher.scala`.
+  
+[![scastie](/resources/img/blog/zinc-0.13-scalatest.gif)](/resources/img/blog/zinc-0.13-scalatest.gif)
+  
+With the previous incremental compiler, the compilation takes 21 seconds in an
+already hot compiler.
+  
+From a method addition, a pretty common operation in any codebase, Zinc 0.13.x
+adds all the users of `AndHaveWord` and recompiles them. These "users" are for
```

Comment:
All of the "users"? This can't be the correct term... Why don't you just say "source files that define a class that reference `AndHaveWord`"? It's a lot clearer.